sdk-playground/packages/client/src/pages/OpenExternalLink.tsx

```diff
@@ -1,21 +1,52 @@
-import discordSdk from '../discordSdk';
+import * as React from "react";
+import discordSdk from "../discordSdk";
+
+enum LinkStatus {
+  NOT_CLICKED = "NOT_CLICKED",
+  CLICKED = "CLICKED",
+  NOT_OPENED = "NOT_OPENED",
+  OPENED = "OPENED",
+}
 
 // Note: we're still using the anchor tag, to ensure standard accessibility UX
 export default function OpenExternalLink() {
+  const [linkStatus, setLinkStatus] = React.useState(LinkStatus.NOT_CLICKED);
+
+  async function handleLinkClicked(
+    e: React.MouseEvent<HTMLAnchorElement, MouseEvent>
+  ) {
+    e.preventDefault();
+    setLinkStatus(LinkStatus.CLICKED);
+    const { opened } = await discordSdk.commands.openExternalLink({
```

Comment:
If the return value here changed, can you also make sure it's reflected in discord-api-docs?

Reply:
https://github.com/discord/discord-api-docs/pull/7273 👍🏻